src/es_client/query.py

```diff
@@ -86,6 +86,11 @@ def search(params, meta):
 
     headers = {'Content-Type': 'application/json'}
 
+    # Add Authorization header if Elasticsearch auth is configured
+    auth_header = get_elasticsearch_auth_header()
+    if auth_header:
+        headers['Authorization'] = auth_header
+
```

Comment:
This appears to be untested

Reply:
Should we do this in the next PR where we add auth to the docker compose?